src/VirtoCommerce.SimpleExportImportModule.Core/Models/ImportProductPrice.cs

```diff
@@ -0,0 +1,16 @@
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.PricingModule.Core.Model;
+
+namespace VirtoCommerce.SimpleExportImportModule.Core.Models
+{
+    public class ImportProductPrice
+    {
+        public string ProductId { get; set; }
+
+        public string ProductCode { get; set; }
```

Comment:
Can we use Sku name please to stop bad practice on Platform to name it differently than business